test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
This is the comment I meant: 

https://github.com/strongloop/strong-remoting/blob/master/lib/shared-class.js#L64

And I agree that we need to preserve this. We should also have unit tests for it. Is it easy to achieve, to test for a full app scenario?